core/services/relay/evm/evmtesting/run_tests.go

```diff
@@ -21,6 +21,17 @@ import (
 	. "github.com/smartcontractkit/chainlink-common/pkg/types/interfacetests" //nolint common practice to import test mods with .
 )
 
+const (
+	QueryKeyFilterOnDataWordsWithValueComparator                             = "Filtering can be done on data words using value comparator"
+	QueryKeyOnDataWordsWithValueComparatorOnNestedField                      = "Filtering can be done on data words using value comparator on a nested field"
+	QueryKeyFilterOnDataWordsWithValueComparatorOnDynamicField               = "Filtering can be done on data words using value comparator on field that follows a dynamic field"
+	QueryKeyFilteringOnDataWordsUsingValueComparatorsOnFieldsWithManualIndex = "Filtering can be done on data words using value comparators on fields that require manual index input"
+	DynamicTypedTopicsFilterAndCorrectReturn                                 = "Dynamically typed topics can be used to filter and have type correct in return"
+	MultipleTopicCanFilterTogether                                           = "Multiple topics can filter together"
+	FilteringCanBeDoneOnHashedIndexedTopics                                  = "Filtering can be done on indexed topics that get hashed"
+	BindReturnsErrorOnMissingContractAtAddress                               = "Bind returns error on missing contract at address"
```

Comment:
This names are pretty hard to read. What is the advantage of using constants over inline literals?